source/slang/hlsl.meta.slang

```diff
@@ -1064,7 +1064,7 @@ extension _Texture<T,Shape,isArray,isMS,sampleCount,0,isShadow,1,format>
     [ForceInline]
     __glsl_extension(GL_ARB_sparse_texture_clamp)
     [require(cpp_glsl_hlsl_metal_spirv, texture_sm_4_1_clamp_fragment)]
-    T Sample(vector<float, Shape.dimensions+isArray> location, vector<int, Shape.planeDimensions> offset, float clamp)
+    T Sample(vector<float, Shape.dimensions+isArray> location, constexpr vector<int, Shape.planeDimensions> offset, float clamp)
```

Comment:
why do we need to add constexpr here?

Reply:
OK, this is fine for now, but I think we need to remove all these constexpr modifiers on sample methods because it is now allowed for offset to be non-constexpr on vulkan and d3d. This can be for another day.